supplier/migrations/0001_initial.py

```diff
@@ -0,0 +1,41 @@
+import django.core.validators
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
```

Comment:
doesn't it depend on User?

Reply:
the user class was declared in the same module as supplier class, i assume thats why it didnt automatically wrote User as dependency.

Reply:
Is it the same for the future Client? will all three be on the same module?

Reply:
there will be two modules since we separated the models by apps, user will be on both supplier and client modules.